Event Tester [DOC-1030] (#7454)
segmentio/segment-docs: src/connections/test-connections.md

The Event Tester sends a real event that appears in your end tool alongside your existing data. 

### Using the Event Tester

> info ""
> The event tester only tests the enabled mappings for the destination. 

To use the Event Tester: 
1. Navigate to **Connections > Destinations** and select your destination.
2. Click the **Event Tester** tab. 
3. Select the type of test event. You can choose from: Track, Identify, Page, Screen, Group. 
4. Enter your test event payload. You can type in your own event or choose from **Load event from source** or **Generate sample event**.
   * **Load event from source**: Segment loads an event based on your source. 
   * **Generate sample event**: Segment generates a sample event for you. 
5. Click **Send test event to destination**. 
  

If your test event successfully sends to the destination, you can see in the **View test outcome** section:
* The request, response, and status for each API call 
* How many of your mappings matched
* The total number of API calls that were made as one test event can result in multiple API calls
* Which mappings were successful and which ones failed
* The destination's API endpoint used to make the request

> ![Screenshot of the Event Tester with a Track test event that resulted in 4 API calls](images/event-tester-2025.png)

Reviewer: Loving this alt text!

---

> Segment's Event Tester enables you to test your connections between Segment and your destination. You can inspect both the request sent from Segment and the response you receive back from the destination. You can use the Event Tester to: 

Reviewer 2: ```suggestion
Segment's Event Tester enables you to test your connections between Segment and your destination. You can inspect both the request sent from Segment and the response you receive back from the destination. You can use the Event Tester to ensure: 
```

> *   ensure an event successfully arrives to a specific destination
> *   ensure your new destination is configured correctly

Reviewer 2: ```suggestion
*   An event successfully arrives to a specific destination
*   Your new destination is configured correctly
```

> The Event Tester sends a real event that will appear in your end tool alongside your existing data. 

Reviewer 2: ```suggestion
The Event Tester sends a real event that appears in your end tool alongside your existing data. 
```

> info ""
> This feature is only available for server-side, [cloud-mode](/docs/connections/destinations/#connection-modes) integrations. It doesn't work for client-side, [device-mode](/docs/connections/destinations/#connection-modes) integrations. 

## How it works

> info ""
> You must have write access in your Segment workspace to use the Event Tester. 

> warning ""
> Make sure you have mappings enabled for your destination. The event tester only tests the enabled mappings for the destination. 

To use the Event Tester: 
1. Navigate to **Connections > Destinations** and select your destination.
2. Click the **Event Tester** tab. 

> 3. Select the type of test event. You can choose from: Track, Identify, Page, Screen, Group. 

Reviewer 2: ```suggestion
3. Select the type of test event. You can choose from the following event types: 
     * Track
     * Identify
     * Page
     * Screen
     * Group 
```
